SPECS/kata-containers-cc/CVE-2025-4574.patch

```diff
@@ -0,0 +1,35 @@
+From 804b30dde9c5e95b794e08bd01c26464ede4933c Mon Sep 17 00:00:00 2001
```

Comment:
This should be fixed upstream first if not yet done, then we can backport this into our fork or consume the fix through rebasing our sources against a new release

Reply:
Looks like this is already fixed upstream resp. in our fork. For:
- kata: The 3.18.0 release uses crossbeam-channel with 0.5.15
- kata-cc: The 3.15.0 release still uses crossbeam-channel 0.5.14. Here, we should extend the release/3.15.0 branch to update the rust crate reference